src/main/java/com/bmc/truesight/saas/meter/client/model/Measure.java

```diff
@@ -22,13 +22,30 @@ public Instant timestamp() {
     }
     public abstract Optional<String> source();
 
+    /**
+     * Creates a Measure using the required parameters: name and value.
+     * The timestamp for this Measure defaults to <a href="https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now--">Instant.now()</a>.
+     * The source of the event will be the Fully Qualified Domain Name of the meter host.
+     * @param name  the name of the Metric displayed in Pulse (must not be greater than 100 characters in length).
+     * @param value the value for this Metric.
+     * @return an Immutable Measure with the name and value given.
+     */
     public static Measure of(String name, double value) {
         return ImmutableMeasure.builder()
                 .name(name)
                 .value(value)
                 .build();
     }
 
+    /**
+     * Creates a Measure using the required parameters: name, value and timestamp.  The source of the event will be the
+     * Fully Qualified Domain Name of the meter host.
```

Comment:
I think (not 100% sure) that the source will end up being whatever is configured as the meter name, which may or may not be the FQDN